scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -358,11 +358,17 @@ object BigQueryTable {
 /**
  * Get an IO for a BigQuery table using the storage API.
  */
-final case class BigQueryStorage(table: Table) extends BigQueryIO[TableRow] {
+final case class BigQueryStorage(table: Table, selectedFields: List[String], rowRestriction: String)
+  extends BigQueryIO[TableRow] {
   override type ReadP = BigQueryStorage.ReadParam
   override type WriteP = Nothing // ReadOnly
 
-  override def testId: String = s"BigQueryIO(${table.spec})"
+  override def testId: String =
+    s"""
```

Comment:
as we talked this can follow the same pattern as the others

```scala
s"""BigQueryIO(${table.spec}, List(${selectedFields.mkString(",")), $rowRestriction)"""
```

Reply:
done